akka-stream-tests/src/test/scala/akka/stream/scaladsl/BidiFlowSpec.scala

```diff
@@ -20,26 +18,22 @@ class BidiFlowSpec extends AkkaSpec with ConversionCheckedTripleEquals {
 
   implicit val mat = ActorMaterializer()
 
-  val bidi = BidiFlow() { b ⇒
-    val top = b.add(Flow[Int].map(x ⇒ x.toLong + 2).withAttributes(name("top")))
-    val bottom = b.add(Flow[ByteString].map(_.decodeString("UTF-8")).withAttributes(name("bottom")))
-    BidiShape(top.inlet, top.outlet, bottom.inlet, bottom.outlet)
-  }
+  val bidi = BidiFlow.fromGraphs(
```

Comment:
@drewhk Yeah, I wasn't 100% happy with it either, but the consistency appealed to me. Any suggested alternatives?